app/js/map/map.controller.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+  'use strict';
+  angular.module('cityInfo.map').controller('MapController', MapController);
```

Comment:
Does this create both a 'cityInfo.map' module and a 'MapController' controller at the same time?
I'm pretty confused what is a setter and getter now.